Sources/SwiftParser/Nominals.swift

```diff
@@ -258,7 +258,12 @@ extension Parser {
           arena: self.arena))
       } while keepGoing != nil && loopProgress.evaluate(currentToken)
     }
-    let rangle = self.consumeAnyToken(remapping: .rightAngle)
+    let rangle: RawTokenSyntax
```

Comment:
What's the precedence of `:` compared to `>`? Could we use `expect` here instead?

Reply:
I don’t think we currently have a way to `expect` the `>` as a token prefix. The problem here is that sometimes the `>` can be part of a bigger token.